include/nuttx/wqueue.h

```diff
@@ -356,8 +359,17 @@ int work_usrstart(void);
  *
  ****************************************************************************/
 
-int work_queue(int qid, FAR struct work_s *work, worker_t worker,
-               FAR void *arg, clock_t delay);
+#if defined(CONFIG_LIB_USRWORK) && !defined(__KERNEL__)
+  int work_queue(int qid, FAR struct work_s *work, worker_t worker,
+                 FAR void *arg, clock_t delay);
+#else
+  int work_queue_nt(int qid, FAR struct work_s *work, worker_t worker,
+                    FAR void *arg, clock_t delay, FAR const char *name_tag);
+
+/* Stringificate the worker name for debug purposes */
```

Comment:
```suggestion
/* Stringify the worker name for debug purposes */
```
nit: I think this is the right verb.

Reply:
Thank you. You are right.